README.rst

```diff
@@ -208,6 +208,34 @@ Non-exhaustive list of Dragonfly modules that should work (with the above change
 - firefox (except save_now command)
 - audacity
 
+Using SSL
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuring SSL:
+TODO
+
+Creating certificates:
+
+First create a certificate authority::
+
+        $ aenea_cert root my_root
+
+This created the files ``my_root.key`` and ``my_root.pem``.
+
+Now create a server certificate::
+
+        $ aenea_cert --root my_root --ip 192.168.1.78 server my_server
```

Comment:
If it works either way we should document that. Some users use Aenea over a local network (all the more reason this crypto is needed!), with physical computers and multiple servers that they switch between using voice commands.

So I could easily see a server being a laptop that's on DHCP. I don't think we need to scrap everything if it won't work for this use case, but if it's easy we should go for it. Another possibility might be -- can you use a DNS name with the cert? For example, instead of an IP could I specify, say, oak.local, and rely on Avahi to provide the name on my local network?

Anyway we don't need this to move forward, but if it's easy to support this case I would prefer to do so.